src/state.tsx

```
@@ -0,0 +1,40 @@
+import { create } from 'zustand'
+
+interface TypedWord {
+    key: string
+    typedWord: string
+}
+
+
+type EnemyStore = {
+  enemies?: EnemyObject[] 
+  getEnemies: CallableFunction
+  enemyFocused?: Boolean
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)